src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+> “Captions not only include dialogue, but identify who is speaking and include non-speech information conveyed through sound, including meaningful sound effects” 
+> -WCAG, Understanding Success Criterion 1.2.2: Captions
+
+Of sites using `<video>` elements, only 0.79% provide closed captions, which we assume based on the presence of the `<track>` element (and which are different from open / burned-in captions). Note that some websites have custom solutions for providing video and audio captions to users. We were unable to detect these custom solutions so the percentage of sites utilizing captions could be higher, but this figure is indicative of how under supported captions are on Web video content. We also cannot assess the quality of the captions detected and whether or not they accurately convey the full meaning of the video they describe.
+
+#### Autoplaying video
+
+It is arguably a disruptive and undesirable user experience to autoplay and loop video on a website for all users. Video can be a resource drain for device batteries
+as well as data, and in some cases video can contain content that is distressing for users, whether by showing disturbing imagery or being used as an attack vector against people prone to seizures. 
+
+For disabled users there are significant barriers caused by autoplaying and looping video. For screen reader users, a video that contains audio will likely disrupt the announcements and lead to confusion. For folks with cognitive disabilities such as ADHD video can be very distracting and interrupt the user’s ability to use and understand the interface. People with vestibular conditions can be dangerously triggered by video as well.
+
+The Web Content Accessibility Guidelines has a criteria [2.2.2 Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html) that requires that any moving, blinking or scrolling content (including video) that plays for longer than 5 seconds have a mechanism to pause, stop, or hide it.
+
+Of the pages with video present, we found that 56.98% of desktop pages and 53.64% of mobile pages have the autoplay attribute meaning that videos play by default. We also found that 58.42% of desktop pages and 52.86% of mobile pages have the loop attribute, which very likely means the video plays indefinitely. Though there could be mechanisms to pause, stop or hide these videos, opting into playing video rather than needing to stop autoplaying and/or looping the video should be the default. These metrics suggest that over half of websites with video could have significant accessibility barriers. 
```

Comment:
```suggestion
Of the pages with video present, we found that 56.98% of desktop pages and 53.64% of mobile pages have the `autoplay` attribute, meaning that videos play by default. We also found that 58.42% of desktop pages and 52.86% of mobile pages have the loop attribute, which very likely means the video plays indefinitely. Though there could be mechanisms to pause, stop or hide these videos, opting into playing video rather than needing to stop autoplaying and/or looping the video should be the default. These metrics suggest that over half of websites with video could have significant accessibility barriers. 
```